homeassistant/components/bloomsky/sensor.py

```diff
@@ -5,53 +5,100 @@
 from homeassistant.const import (
     AREA_SQUARE_METERS,
     CONF_MONITORED_CONDITIONS,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_PRESSURE,
     DEVICE_CLASS_TEMPERATURE,
     ELECTRIC_POTENTIAL_MILLIVOLT,
+    LENGTH_INCHES,
+    LENGTH_MILLIMETERS,
     PERCENTAGE,
+    PRECIPITATION_INCHES_PER_HOUR,
+    PRECIPITATION_MILLIMETERS_PER_HOUR,
     PRESSURE_INHG,
     PRESSURE_MBAR,
+    SPEED_METERS_PER_SECOND,
+    SPEED_MILES_PER_HOUR,
     TEMP_CELSIUS,
     TEMP_FAHRENHEIT,
+    UV_INDEX,
 )
 import homeassistant.helpers.config_validation as cv
 
 from . import DOMAIN
 
+# See http://weatherlution.com/bloomsky-api/
+# http://weatherlution.com/wp-content/uploads/2016/01/v1.6BloomskyDeviceOwnerAPIDocumentationforBusinessOwners.pdf
+
 # These are the available sensors
 SENSOR_TYPES = [
-    "Temperature",
+    # SKY
     "Humidity",
-    "Pressure",
     "Luminance",
-    "UVIndex",
+    "Pressure",
+    "Temperature",
+    "UVIndex",  # Also on storm
     "Voltage",
+    # Storm
+    "24hRain",  # last 24hs
+    "RainDaily",  # 12a-1159p
+    "RainRate",  # last 10m
+    "SustainedWindSpeed",
+    "WindDirection",  # NW, etc
+    "WindGust",
```

Comment:
This extends the monitored conditions of this integration, which is no longer allowed to be extended.

See: <https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

Instead, this integration should be adjusted to remove monitored conditions (or even better: Update to use a config flow as per ADR-0010).

Reply:
Done, removed monitored conditions.

I'd like to delay tackling switching to a config flow for a later PR if at all possible.

Reply:
I added a second commit to make the integration automatically setup its platforms, at which point the only part left in configuration.yaml is:

```yaml
bloomsky:
  api_key: foobar
```

and everything else gets set up automatically.

Happy to merge squash this commit into the first one if this is desirable (or drop the commit if undesired)